src/Entity.php

```diff
@@ -2047,500 +2004,43 @@ public static function showHelpdeskOptions(Entity $entity)
-
-        echo "<tr class='tab_bg_1'>" .
-         "<td>" . __('Automatic closing of solved tickets after');
-
-       //Check if crontask is disabled
         $crontask = new CronTask();
-        $criteria = [
-            'itemtype'  => 'Ticket',
-            'name'      => 'closeticket',
-            'state'     => CronTask::STATE_DISABLE
-        ];
-        if ($crontask->getFromDBByCrit($criteria)) {
-            echo "<br/><strong>" . __('Close ticket action is disabled.') . "</strong>";
-        }
-
-        echo "</td>";
-        echo "<td>";
-        $autoclose = [self::CONFIG_PARENT => __('Inheritance of the parent entity'),
-            self::CONFIG_NEVER  => __('Never'),
-            0                   => __('Immediatly')
-        ];
-        if ($ID == 0) {
-            unset($autoclose[self::CONFIG_PARENT]);
-        }
-
-        Dropdown::showNumber(
-            'autoclose_delay',
-            ['value' => $entity->fields['autoclose_delay'],
-                'min'   => 1,
-                'max'   => 99,
-                'step'  => 1,
-                'toadd' => $autoclose,
-                'unit'  => 'day'
-            ]
-        );
-
-        if ($entity->fields['autoclose_delay'] == self::CONFIG_PARENT) {
-            $autoclose_mode = self::getUsedConfig(
-                'autoclose_delay',
-                $entity->fields['entities_id'],
-                '',
-                self::CONFIG_NEVER
-            );
-
-            if ($autoclose_mode >= 0) {
-                self::inheritedValue(sprintf(_n('%d day', '%d days', $autoclose_mode), $autoclose_mode), true);
-            } else {
-                self::inheritedValue($autoclose[$autoclose_mode], true);
-            }
-        }
-        echo "<td>" . __('Automatic purge of closed tickets after');
-
-       //Check if crontask is disabled
-        $crontask = new CronTask();
-        $criteria = [
-            'itemtype'  => 'Ticket',
-            'name'      => 'purgeticket',
-            'state'     => CronTask::STATE_DISABLE
-        ];
-        if ($crontask->getFromDBByCrit($criteria)) {
-            echo "<br/><strong>" . __('Purge ticket action is disabled.') . "</strong>";
-        }
-        echo "</td>";
-        echo "<td>";
-        $autopurge = [
-            self::CONFIG_PARENT => __('Inheritance of the parent entity'),
-            self::CONFIG_NEVER  => __('Never')
-        ];
-        if ($ID == 0) {
-            unset($autopurge[self::CONFIG_PARENT]);
-        }
-
-        Dropdown::showNumber(
-            'autopurge_delay',
-            [
-                'value' => $entity->fields['autopurge_delay'],
-                'min'   => 1,
-                'max'   => 3650,
-                'step'  => 1,
-                'toadd' => $autopurge,
-                'unit'  => 'day'
+        $inheritance_labels = [
+            'tickettemplates_id' => $entity->getInheritedLinkedValueBadge(TicketTemplate::class),
```

Comment:
I do not know if nothing, `-----`, or `None` is the best label to display. Anyways, it should be done the same for all options.